### PR TITLE
Jk 2023 08 14 newprayer

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Prayer work records created based on https://github.com/BetaMasaheft/Documentation/issues/2409

### CODE REVIEW

File: new/LIT6855Prayer.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6855Prayer" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Short prayer</title>
```
Comment (thea-m) [inline new/LIT6855Prayer.xml:8]: "en"
Could you give, either very briefly in the title or in the abstract, a short characterisation of the prayer's contents?

File: new/LIT6856PPHemamaBarya.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6856PPHemamaBarya" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer against Barya and Legewon</title>
```
Comment (thea-m) [inline new/LIT6856PPHemamaBarya.xml:8]: Bāryā

File: new/LIT6856PPHemamaBarya.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Prayers"/>
```
Comment (thea-m) [inline new/LIT6856PPHemamaBarya.xml:37]: Magic?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! ChristianContent is reserved for items that do not contain text, it should be ChristianLiterature here (see schema definitions).

File: new/LIT6856PPHemamaBarya.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                        <seg part="I">በስመ <supplied reason="undefined">፡</supplied> አብ፡ 
+                            ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ በእንተ፡ ሕማመ፡ ባርያ፡ ወለ<del rend="effaced">ያ</del>ጌዎን፡ 
+                        </seg> አ<del rend="effaced">ሁ</del>ላሁ፡ <foreign xml:lang="am">ማቴ፡ አልመላክያው፡ 
+                            ኑራተዊ፡ <del rend="effaced">ቋ</del>ቁራ፡ ስስምረጅ፡ ወመርመ፡ የፉሕሙ፡ የሽፋቴ፡</foreign>
+                    </ab>
+                </div>
+                <div type="textpart" xml:lang="gez" subtype="explicit">
+                    <note>The following explicit is taken from <ref type="mss" corresp="BDLaethf25"/>.</note>
+                    <ab>
+                        ንድሃን፡ በሃይለ፡ ዝንቱ፡ አስማቲከ፡ አድኖ፡ ለገብርከ፡ ገብረ፡ እገሌ፡ እምባርያ፡ ወላጌዎን፡
```
Comment (MassimoVilla) [inline new/LIT6856PPHemamaBarya.xml:71]: perhaps አድ`<supplied reason="omitted">ኅ</supplied>`ኖ፡ ?

File: new/LIT6858PPsnakes.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6858PPsnakes" passive="BDLaethf25"/>
+                    <relation name="lawd:hasAttestation" active="LIT6858PPsnakes" passive="ESdd045"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>This is the entire text, which is taken from <ref type="mss" corresp="BDLaethf25"/>.</note>
+                    <ab>
+                        በስመ፡ አብ፡ ወወልድ፡ ወመን፡ ፈስ፡ ቅዱስ፡ ፩ አምላክ፨ አክዮስ፡ አክዮስ፡
```
Comment (MassimoVilla) [inline new/LIT6858PPsnakes.xml:62]: ወመ`<surplus>ን፡</surplus>`ፈስ፡

File: new/LIT6862PPscribe.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+                        <hi rend="rubric">ሰሐል፡ ሰሐል፡ ሰሐል፡ 
+                            ጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸ<del rend="strikethrough">ጸ</del>እዴየ፡ ሰሐል፡ 
+                            ወየማንየ፡ አሕመላምል፡ ለጺሒፈ፡ ብርዕ፡ ወቀለም፡ ሊተ፡ ለገብርከ፡ አፍጌት፡ ወርኃ፡ ደቡብ፡ ወወርኃ፡ 
+                            ደመና፡ <del rend="encircled strikethrough">አክሄል፡ ድሜሄላዊ፡</del> መንጥሔል፡ 
+                            ድሜሄላዊ፡ በከመ፡ የሐውሩ፡ ፀሐይ፡ ወወርኅ፡ በፍጥረተ፡ ትእዛዝከ፡ ከማሆሙ፡ አሩፅ፡ የማነ፡ እዴየ፡ 
+                            ለጋ፡ በስመ፡ <del rend="strikethrough">ለለ</del>ሎኤ፡ ኢላኤ፡ ኢላሄ፡ አሩፅ፡ እዴየ፡ 
+                            ለገቢረ፡ ኵሉ፡ ተግባር፡ በሩፋኤል፡ ብርናዊ፡ በክሳብኤል፡ 
+                            ዘ<del rend="strikethrough">ፈ</del>ያረውፃ፡ ለፀሐይ፡ ከማሁ፡ አሩፅ፡ እዴየ፡ ከመ፡ 
+                            ይጠንቅቅ፡ ለአኂዘ፡ ብርዕ፡ ወእጽሐፍ፡ እበያቲከ፡ ዘእንበለ፡ ሐኬት፡ ኦእግዚእየ፡ ኢየሱስ፡ 
+                            ክርስቶስ፡ አስተ፡ ፋጥን<supplied reason="undefined">፡</supplied>ወ<add place="above">አ</add>ስተ፡
```
Comment (MassimoVilla) [inline new/LIT6862PPscribe.xml:70]: `አስተ<surplus>፡</surplus>ፋጥን<supplied reason="undefined">፡</supplied>` 
`አስተ<surplus>፡</surplus>ቃንዕ፡`

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Jonas, I have just added few suggestions about text encoding. Please feel free to consider them

**Review by thea-m — APPROVED** [review]

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**